app/server/me_api/credit_transfer.py

```diff
@@ -102,6 +103,8 @@ def post(self):
 
         is_sending = post_data.get('is_sending', False)
 
+        method = 'ANY'
```

Comment:
we already have a 'transfer_mode' on credit transfers (which is super neglected and under used) - but i think this should use that...